src/testdir/test_blob.vim

```diff
@@ -318,3 +320,15 @@ endfunc
 func Test_blob_sort()
   call assert_fails('call sort([1.0, 0z11], "f")', 'E975:')
 endfunc
+
+func Test_str2blob()
+  call assert_equal(0z612F622F.63E4B896.E7958C, str2blob('a/b/c世界'))
+  call assert_equal(0z, str2blob(''))
+  call assert_equal(0z30, str2blob(0))
+endfunc
+
+func Test_blob2str()
+  call assert_equal('a/b/c世界', blob2str(0z612F622F.63E4B896.E7958C))
+  call assert_equal('', blob2str(0z))
```

Comment:
According to the help file, `blob2str()` has a second optional argument:
```
blob2str({blob} [, {sep})
```
I don't see a test with the `{sep}` argument.

Also, when `{sep}` is not given, help file says:
```
NUL character is removed from result string when {sep} is omitted.
```
But I don't see that being tested either.